cmd/minishift/cmd/root.go

```diff
@@ -68,7 +72,15 @@ var RootCmd = &cobra.Command{
 	Short: "Minishift is a tool for application development in local OpenShift clusters.",
 	Long:  `Minishift is a command-line tool that provisions and manages single-node OpenShift clusters optimized for development workflows.`,
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
-		var err error
+		var (
+			err          error
+			isFreshStart bool
+		)
+
+		if filehelper.IsEmptyDir(constants.Minipath) {
```

Comment:
@hferentschik , are you saying we should extract out the logic of checking file existance from `IsEmptyDir` ?